### PR TITLE
1st push Dev

### PR DESCRIPTION
**Pull request by MaxAnxxi**
First version of the project done. 

CSS Rules are pretty much cleaned , only having a footer not displayed as we wish

Main components are here : Navbar , Formation , Footer and Lesson

Lesson is getting displayed when we click on a button , since we only have one lesson yet , we're displaying it whenever a button is clicked no matter which one. 

Navbar is fixed on the left for now , willing to add a menu burger later.

### CODE REVIEW

File: client/src/components/formation/Formation.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <div className="buttonFormations">
+      {name.map((element, index) => (
+        <button type="button" key={index.id} onClick={handleClick}>{element.formation}</button>
+      ))}
+      {show ? <Lesson/> : null}
+    </div>
+  );
+}
+Formation.propTypes={
+    name: PropTypes.arrayOf(
```
Comment (wildmatthieu) [inline client/src/components/formation/Formation.jsx:24]: indentation à revoir (2 espaces à la place de 4 espaces)
à répercuter dans le reste du fichier

File: client/src/components/lesson/Lesson.jsx
[the remark this answers is not in the document]
Comment (YohanGH) [reply client/src/components/lesson/Lesson.jsx:1]: Fix 👍

File: client/src/components/footer/FooterStyle.module.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* FooterStyle.module.css */
+.containerFooter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  font-family: "Helvetica", sans-serif;
+  font-size: 1rem;
+  /* margin-top: auto; */
```
[the remark this answers is not in the document]
Comment (YohanGH) [reply client/src/components/footer/FooterStyle.module.css:9]: fix 👍

File: client/src/App.jsx [hunk 3]
```diff
@@ -1,89 +1,28 @@
...
+import BannerLogo from "./components/banner_logo/BannerLogo";
 
-function App() {
-  return (
-    <main className="container">
-      <header>
-        <h1 className="logo">Harmonia</h1>
-      </header>
+const name = [
+	{ formation: "HTML" },
```
[the remark this answers is not in the document]
Comment (YohanGH) [reply client/src/App.jsx:8]: fix 👍

File: client/src/components/formation/Formation.jsx (hunk 1 above)
Comment (YohanGH) [reply client/src/components/formation/Formation.jsx:24]: fix 👍

File: client/src/components/formation/Formation.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  function handleClick(){
+    setShow(!show);
+  }
+
+    return (
+    <div className="buttonFormations">
+      {name.map((element, index) => (
+        <button type="button" key={index.id} onClick={handleClick}>{element.formation}</button>
+      ))}
+      {show ? <Lesson/> : null}
```
[the remark this answers is not in the document]
Comment (YohanGH) [reply client/src/components/formation/Formation.jsx:19]: Fix 👍

**Pull request merged**